### ISSUE TITLE
How stop the call when caller ends call?

### ISSUE THREAD

**Issue by ScottLee97** [flutter-webrtc/callkeep#95]
A user calls someone and the backend would send out an FCM data-only message to the receiver which triggers the call. But what if the caller ends the call before the receiver picks up the call? 
Is there a way to end the call on the receiver's side when the caller ends call?

**Comment by efraespada** [comment]
No, you must control it by yourself.

I start calls with FCM data push messages (the app can be closed) and end them with Firestore Database (app is open and listening call-room changes).

**Comment by ScottLee97** [comment]
Is it possible to start and end the call when the app is in the background? In other words, can the listening of call-room changes be done even when the app is in the background??

**Comment by efraespada** [comment]
No, when the app is terminated or background (even iOS) your listeners will stop receiving changes. Use push data messages to do that.
